core/chains/evm/config/config_test.go

```diff
@@ -229,11 +231,18 @@ func TestChainScopedConfig_Profiles(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
```

Comment:
This might race, suggested fix below:

```suggestion
	for _, test := range tests {
	        tt := test 
		t.Run(tt.name, func(t *testing.T) {
			t.Parallel()
```

Reply:
Fixed